src/assets/js/views/taskView.js

```diff
@@ -0,0 +1,78 @@
+import getElement from '../helpers/getElement';
+import createElement from '../helpers/createElement';
+
+export default class TaskView {
+  constructor() {
+    this.main = getElement('.main');
+
+    this.headerDiv = createElement('div', { class: 'header-div d-flex' });
+    this.heading = createElement('h2', { }, "Today's To Do");
+    this.refreshDiv = createElement('div', { class: 'refresh-div' });
+    this.refreshIcon = createElement('span', { class: 'material-icons refresh-icon' }, 'autorenew');
+    this.pendingCount = createElement('span', { class: 'pending-count d-flex' });
```

Comment:
CreateElement can be made DRY by grouping them all in a function and using the function instead